openml-lightgbm/lightgbm-builder/pom.xml

```diff
@@ -15,30 +15,30 @@
 
     <groupId>com.feedzai.openml.lightgbm</groupId>
     <artifactId>lightgbm-lib</artifactId>
-    <version>2.3.190</version>
+    <version>3.0.0</version>
 
     <packaging>jar</packaging>
     <name>Openml LightGBM lib</name>
     <description>
         LightGBM build for Java generated with make-lightgbm.
 
-        Build created with command: `bash make.sh fc79b366153e8aa5cf2996b4b753e28e5a75e878 2.3.190`
+        Build created with command: `bash make.sh v3.0.0 3.0.0`
 
         ---
 
         Build info:
 
         LIGHTGBM_REPO_URL=https://github.com/microsoft/LightGBM
-        LIGHTGBM_VERSION=fc79b366153e8aa5cf2996b4b753e28e5a75e878
-        LIGHTGBM_COMMIT=fc79b366153e8aa5cf2996b4b753e28e5a75e878
-        PACKAGE_TIMESTAMP=20/08/21 07:43:12
+        LIGHTGBM_VERSION=v3.0.0
+        LIGHTGBM_COMMIT=7e11d4aeabd4a39ffa4afb382299c6d00ddf01e7
+        PACKAGE_TIMESTAMP=20/09/01 18:20:49
```

Comment:
why 20 and not 2020?

Reply:
Just what datetime from linux generated.